generators/client/templates/react/src/main/webapp/app/modules/administration/configuration/configuration.tsx

```diff
@@ -40,77 +50,100 @@ export class ConfigurationPage extends React.Component<IConfigurationPageProps,
 
   filterFn = prefix => prefix.toUpperCase().includes(this.state.filter.toUpperCase());
 
+  sortPrefix = (configProps, rev) => {
+    const { configuration } = this.props;
+    if (configuration && configuration.configProps) {
+      const listSorted = sortBy(configProps, [ prop => {
```

Comment:
`const listSorted = sortBy(configProps, [ prop => configuration.configProps[prop].prefix ]);` will be more elegant